14/umbraco-cms/tutorials/custom-error-page.md

```diff
@@ -36,18 +36,19 @@ This method will use a 404 page created via the backoffice.
 
 1. Navigate to the **Settings** section.
 2. Create a new "_Document Type with Template_".
-3. Name the Document Type **404**.
+3. Name the Document Type **404 Page**.
+   1. Note: By default, Umbraco will generate the alias for this DocType as simply "Page," omitting the "404." You may want to manually adjust the alias to "404Page."
```

Comment:
How about adding a new step for actually updating the alias instead?
Like, "`4. Change the alias to **Page404**`" or something similar.

What do you think?

Reply:
What if I told you that I completely agree, on re-reading this the second time though? :)

I added a step 4 and updated the rest of the sequence. I also left a note on step 4 to give the "why." It feels awkward maybe, but I think it's important to tell people why they need to do this.

Submitted a change, LMK what you think!